docker/images/node/Dockerfile

```diff
@@ -18,9 +18,9 @@ RUN go build -mod=vendor -tags netgo -ldflags="-w -s" \
     go build  -mod=vendor -ldflags="-w -s" -o skywire-cli ./cmd/skywire-cli	&&\
     go build  -mod=vendor -ldflags="-w -s" -o ./apps/skychat.v1.0 ./cmd/apps/skychat	&&\
 	  go build  -mod=vendor -ldflags="-w -s" -o ./apps/helloworld.v1.0 ./cmd/apps/helloworld &&\
-	  go build  -mod=vendor -ldflags="-w -s" -o ./apps/socksproxy.v1.0 ./cmd/apps/therealproxy &&\
-	  go build  -mod=vendor -ldflags="-w -s" -o ./apps/socksproxy-client.v1.0  ./cmd/apps/therealproxy-client
-
+	  go build  -mod=vendor -ldflags="-w -s" -o ./apps/socksproxy.v1.0 ./cmd/apps/skysocks &&\
+	  go build  -mod=vendor -ldflags="-w -s" -o ./apps/socksproxy-client.v1.0  ./cmd/apps/skysocks-client &&\
```

Comment:
Why do we need `&&\` at the end of the line?

Reply:
No we don't; it's been removed.